model/Core/Classes/NamespaceMap.md

```diff
@@ -14,40 +14,24 @@ of ElementIDs to be used in SPDX content serialization in order to provide a mor
 
-If utilized the namespace map set of prefixes and namespaces MUST be implemented in a given serialization 
-form (e.g., json-ld or xml) as specified in the binding rules specification for that serialization and 
-utilizing the appropriate inherent or custom specified mechanism for that serialization.
-The namespace map itself is also conveyed as native SPDX content to support clarity, transparency and 
-consistency independent of any particular serialization form.
-
-A given serialization payload (whether file or streaming) MUST NOT contain multiple namespace maps with conflicting mappings.
-Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content unless the single 
-serialization payload is intended to contain multiple peer collections each with their own namespace map.
-In this case the serializer MUST do one of the following:
-1) Diverge from the exact provided namespace mappings and utilize deconflicted mappings where appropriate.
-   This could be fully new prefix mappings as if the suggested namespace maps had not been provided.
-3) Use full ElementID IRIs instead of prefixed ElementID IRIs.
-
+If a serialization format supports prefixes or namespaces (e.g., JSON-LD context or XML XSD namespace), the namespace map
+must be represented in that format "native" to the serialization.
+If the serialization format does not support prefixes, then the full URI's for the elements must be used and the namespace map will not be preserved.
+Any custom serialization format SHOULD implement namespaces in order to preserve the namespace map.
 
 Namespace maps support a variety of relevant use cases such as:
 
-1) An SPDX content producer wishing to provide clarity of their serialization of an SPDX 2.X simple style collection where all content is newly minted and a single prefix-namespace is used.
-2) An SPDX content producer wishing to maintain consistent prefix use and understanding across multiple different serializations of the produced content.
-   For example, an SBOM producer wishes to share/publish the SBOM as json-ld, XML and YAML. The producer can specify the preferred prefix mappings they plan to use in all serializations, maintain usage consistency themselves across those serializations and provide clarity to the consumer of the SBOM in any of those serializations independent of which serialization form was received.
+1) An SPDX content producer wishing to provide clarity of their serialization of an SPDX 2.X simple style collection where all content is newly minted and a single prefix-namespace is used.  The consumer of SPDX content wishes to preserve the name space mapping provided by such a producer.  In this case, the consumer would record the namespace map prefixes in the NamespaceMap such that subsequent serializations could reproduce the prefixes / namespaces in the native serialization format.
```

Comment:
> The consumer of SPDX content wishes to preserve the name space mapping provided by such a producer.  In this case, the consumer would record the namespace map prefixes in the NamespaceMap such that subsequent serializations could reproduce the prefixes / namespaces in the native serialization format.

It is unclear exactly what is meant by the consumer to "record" the namespace map. Is this saying they should store it as SPDX content along with their other SPDX content but never serializa it out?
It is also unclear what is meant by "subsequent serialization". Does this many a single reserialization of the content, ANY future serializations of the entire content or ANY future serializations of portions of this content?

Reply:
> It is unclear exactly what is meant by the consumer to "record" the namespace map. Is this saying they should store it as SPDX content along with their other SPDX content but never serializa it out?

This was clarified by the correction above

> t is also unclear what is meant by "subsequent serialization". Does this many a single reserialization of the content, ANY future serializations of the entire content or ANY future serializations of portions of this content?

Both - any future serializations in full or part